pkgs/by-name/ge/gerbolyze/package.nix

```
@@ -0,0 +1,95 @@
+{ lib
+, stdenv
+, python3Packages
+, fetchFromGitHub
+, resvg
+}:
+
+let
+  version = "3.1.7";
+  src = fetchFromGitHub {
+    owner = "jaseg";
+    repo = "gerbolyze";
+    rev = "v${version}";
+    sha256 = "sha256-0iTelSlUJUafclRowwsUAoO44nc/AXaOKXnZKfKOIaE=";
```

Change this to `hash`, the prefix of the hash string already implies the hash algorithm
```suggestion
    hash = "sha256-0iTelSlUJUafclRowwsUAoO44nc/AXaOKXnZKfKOIaE=";
```

---

Whoops, started with some code that had the old way of doing things.